src/redux/books/books.js

```diff
@@ -1,28 +1,51 @@
 const INIT_CONST = 'bookstore/books/';
-// const LOAD_BOOKS = `${INIT_CONST}LOAD_BOOKS`;
+const LOAD_BOOKS = `${INIT_CONST}LOAD_BOOKS`;
 const ADD_BOOK = `${INIT_CONST}ADD_BOOK`;
 const REMOVE_BOOK = `${INIT_CONST}REMOVE_BOOK`;
-
+const BOOK_API = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/hi5j493lEPZn8qT2kBPM/';
 const initState = {
   books: [],
+  savedBooks: [],
   counter: 1,
 };
-const remove = (tab = [], id) => {
+/* const remove = (tab = [], id) => {
   const newBooks = tab.filter((book) => book.id !== id);
   return newBooks;
+}; */
```

Comment:
- [ ] It would be better if you don't have commented code. Kindly delete this commented code. 

😄  🛠️